code/script.js

```diff
@@ -1,9 +1,15 @@
-// A function that adds and remove the class "active" on the section you click on.
-// We haven't really talked about ´this´ yet, but we will... ;)
-// console.log(this) to see how it works
+// Hopefully I understood what this does. Still unsure. 
 function toggle() {
-  this.classList.toggle("active")
+  this.classList.toggle('active')
 }
 
-// Selects an HTML element, and calls a function which will be executed when the element is clicked.
-document.getElementById("section1").onclick = toggle
+// Still kind of confused about how this works
+document.getElementById('section1').onclick = toggle
+document.getElementById('section2').onclick = toggle
+document.getElementById('section3').onclick = toggle
+document.getElementById('section4').onclick = toggle
+document.getElementById('section5').onclick = toggle
+document.getElementById('order').onclick = toggle
```

Comment:
This is totally different from what I did but I understand how it works and its interesting seeing you doing something totally different from me but gaining the same result!